fairer-iOS/fairer-iOS/Screens/Setting/ManageHouse/ManageHouseViewController.swift

```diff
@@ -195,3 +197,22 @@ extension ManageHouseViewController: UITableViewDelegate, UITableViewDataSource
         }
     }
 }
+
+extension ManageHouseViewController {
+    func postLeaveTeam() {
+        NetworkService.shared.teams.postLeaveTeam { [weak self] result in
+            switch result {
+            case .success(_):
+                let pushVC = GroupMainViewController()
+                if let navigationController = self?.navigationController {
+                    navigationController.pushViewController(pushVC, animated: true)
+                    navigationController.setViewControllers([pushVC], animated: true)
+                }
```

Comment:
p1;
올려주신 거 영상에서 navigation될 때 navbar 가 생성되었다가 다시 사라지는 모습이 보입니다!
push 후 stack 비우는 과정이 문제인데요!
```suggestion
                if let navigationController = self?.navigationController {
                    navigationController.setViewControllers([pushVC], animated: true)
                }
```
그냥 뷰컨 스택을 비우고 pushVC로 바꿔주시면 문제가 해결됩니당!

Reply:
아하 꼼꼼하시군요 역시.......